django/template/base.py

```diff
@@ -880,6 +880,10 @@ def _resolve_lookup(self, context):
         try:  # catch-all for silent variable failures
             for bit in self.lookups:
                 try:  # dictionary lookup
+                    # Only allow if the metaclass implements __getitem__.
+                    # See https://docs.python.org/3/reference/datamodel.html#classgetitem-versus-getitem
```

Comment:
```suggestion
                    # Only allow if the metaclass implements __getitem__. See
                    # https://docs.python.org/3/reference/datamodel.html#classgetitem-versus-getitem
```
(possibly there is a special rule about link comments but I guess unlikely)

Reply:
I have taken the link out again and kept a "lose" reference

Reply:
Apparently it does work if it's a line on it's own